var/spack/repos/builtin/packages/metis/package.py

```diff
@@ -207,8 +208,9 @@ def install(self, spec, prefix):
             for o in rpath_options:
                 options.remove(o)
         if '+debug' in spec:
-            options.extend(['-DDEBUG:BOOL=ON',
-                            '-DCMAKE_BUILD_TYPE:STRING=Debug'])
+            if build_type != 'Debug':
+                raise InstallError("Found +debug but build_type!=Debug.")
```

Comment:
actually, i meant to get rid of `+debug` altogether and use `build_type` in `4.x` installation, but only if it's set to `Debug` or `Release`.

Reply:
Ah - I totally misunderstood. I'll make this change right now.